### ISSUE TITLE
Evaluating Poisson Reconstruction at Specified Query Points

### ISSUE THREAD

**Issue by danielklaw** [mkazhdan/PoissonRecon#115]
Is it possible to compute the Poisson Surface Reconstruction at user-specified query locations within the point cloud? Thank you!

**Comment by mkazhdan** [comment]
Yes. In FEMTree.h there is a class "MultiThreadedWeightEvaluator" that is designed precisely for that.
For an example of usage, see lines 651-668 in PoissonRecon.cpp.
The one caveat is that things have already been mapped to the unit-cube domain, so if your sample point is expressed in the coordinate system of the point cloud, you will need to transform it into the unit cube's system.
The inverse transform, iXForm is defined in Line 451 of PoissonRecon.cpp.
